internal/resolution/resolve.go

```diff
@@ -104,6 +114,92 @@ func Resolve(ctx context.Context, cl client.ResolutionClient, m manifest.Manifes
 	return result, nil
 }
 
+func resolvePostProcess(ctx context.Context, cl client.ResolutionClient, m manifest.Manifest, opts ResolveOpts, graph *resolve.Graph) (*resolve.Graph, error) {
+	if m.System() == resolve.Maven && opts.MavenManagement {
+		// Add a node & edge for each dependency in dependencyManagement that doesn't already appear in the resolved graph
+		manifestSpecific, ok := m.EcosystemSpecific.(manifest.MavenManifestSpecific)
+		if !ok {
+			return graph, errors.New("invalid MavenManifestSpecific data")
+		}
+
+		// Search through OriginalRequirements management dependencies in this pom only (not parents).
+		for _, req := range manifestSpecific.OriginalRequirements {
+			if req.Origin != mavenutil.OriginManagement {
```

Comment:
how about dependency management in profiles?

Reply:
I'm going to put this as a TODO, since writing to profiles is a bit iffy - https://github.com/google/osv-scanner/issues/1239#issuecomment-2345028438

Reply:
TODO may also include the dependencies in non-activated profiles? not only the dependency management